### PR TITLE
Update sessions.md

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/program/apis/sessions.md [hunk 1]
```diff
@@ -85,6 +85,10 @@ If you manually manage sessions, each client must send at least one heartbeat wi
 The [Session Management API](https://pkg.go.dev/go.viam.com/rdk/session) is not currently provided in the Python or TypeScript SDKs.
 Use the Go Client SDK instead.
 
+{{< alert title="Tip" color="tip" >}}
+If you are looking to implement session management yourself only to increase the session window, you can increase the session window instead, by ...
```
Comment (npentrel) [inline docs/program/apis/sessions.md:89]: which parameter does this?

Comment (maximpertsov) [reply docs/program/apis/sessions.md:89]: note that the default is 2s

File: docs/program/apis/sessions.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The period of time during which a client is connected to a robot is called a _se
 
 _Session management_ is a presence mechanism that allows you to manage the clients that are authenticated and communicating with a robot's `viam-server` instance.
 
-As a safety precaution, the default session management configuration ensures that a robot only moves when a client is actively connected.
+It also functions as a safety precaution: the default session management configuration ensures that a robot only moves when a client is actively connected and stops any components that remain running when a client disconnects.
```
Comment (maximpertsov) [inline docs/program/apis/sessions.md:26]: I would say that safety is the _primary_ function of the session manager. it doesn't exist for any other reason at the moment.

File: docs/program/apis/sessions.md (hunk 1 above)
Comment (npentrel) [reply docs/program/apis/sessions.md]: thanks for the screenshot!

**Review by maximpertsov — APPROVED** [review]

**Pull request merged**